dev_cmds/setup-dev.ts

```diff
@@ -0,0 +1,52 @@
+import { execPiped, runAsyncMain } from "devcmd";
+import { YARN_COMMAND } from "./utils/commands";
+import { devcmdPackageDir, devPath, singlePackageJsonExampleDir } from "./utils/paths";
+import fs from "fs-extra";
+
+async function main() {
+
+  if(await fs.pathExists(devPath)) {
+    process.stderr.write('A development folder already exists!\n')
+    process.exit(1)
```

Comment:
btw, you can just throw an error instead of logging & exiting. `runAsyncMain` will catch, log and abort :)